spec/controllers/api/scores_controller_spec.rb

```diff
@@ -13,13 +13,21 @@
 
   describe 'GET feed' do
     it 'should return the token if valid username/password' do
+      get api_feed_path
+      expect(response).to have_http_status(:ok)
+    it 'should return 25 scores on the feed' do
+      30.times do
+        create(:score, user: @user1, total_score: 79, played_at: '2021-05-20')
+      end
```

Comment:
Automated Tests (`rspec`) are failing because of **syntax errors**. Figure out what is wrong in this file.